services/cdap-standalone-docker.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cdap-standalone",
+  "name": "cdap-standalone-docker",
   "description": "CDAP SDK (Standalone) running in a Docker container (caskdata/cdap-standalone)",
```

Comment:
perhaps (to be consistent) this should be:
```
"description": "CDAP Standalone running in a Docker container (caskdata/cdap-standalone)",
```
as the SDK doesn't "run", and we should distinguish the SDK from the running server.